frontend/src/scenes/dashboard/Dashboards.tsx

```diff
@@ -187,7 +189,15 @@ export function Dashboards(): JSX.Element {
             <PageHeader
                 title="Dashboards"
                 buttons={
-                    <LemonButton data-attr={'new-dashboard'} onClick={showNewDashboardModal} type="primary">
+                    <LemonButton
+                        data-attr={'new-dashboard'}
+                        data-tooltip="experiment-dashboards-product-tour"
+                        onClick={() => {
+                            closePrompts()
+                            showNewDashboardModal()
```

Comment:
This is gonna be fun... We'll have to see how it goes with the trial but perhaps it shouldn't be possible to do anything other than interact with the tooltips as there are lots of other places that create modals...